pkg/commands/configstore/update.go

```diff
@@ -0,0 +1,69 @@
+package configstore
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/cmd"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/global"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v7/fastly"
+)
+
+// NewUpdateCommand returns a usable command registered under the parent.
+func NewUpdateCommand(parent cmd.Registerer, g *global.Data, m manifest.Data) *UpdateCommand {
+	c := UpdateCommand{
+		Base: cmd.Base{
+			Globals: g,
+		},
+		manifest: m,
+	}
+
+	c.CmdClause = parent.Command("update", "Update a config store")
+
+	// Required.
+	c.RegisterFlag(cmd.StoreIDFlag(&c.input.ID)) // --store-id
+	c.RegisterFlag(cmd.StringFlagOpts{
+		Name:        "name",
+		Short:       'n',
+		Description: "New name for the config store",
+		Dst:         &c.input.Name,
+		Required:    true,
+	})
```

Comment:
As mentioned elsewhere for the optional flags, can we ensure these required flags are ordered alphabetically so searching for flags is easier. Keeps the flag behaviour consistent across all commands.

Reply:
Also, I'll stop mentioning this for now and let you check all instances of this 🙂

Reply:
ad3ce1c